tests/testsuite/kubevirtresource.go

```diff
@@ -75,6 +75,11 @@ func AdjustKubeVirtResource() {
 		},
 	}}
 
+	// Add the DownwardMetrics configuration, it will avoid to make some test to run on serial
+	if kv.Spec.DownwardMetrics == nil {
+		kv.Spec.DownwardMetrics = &v1.DownwardMetricsConfiguration{}
```

Comment:
This enables the feature which should be off by default for everyone else? I don't think this is good practice.

Reply:
If we don't enable it by default, live migration downwardMetric tests will need to run on serial because we are updating the KubeVirt CR. In the past, in order to allow these test to run on parallel, we enabled the downwardMetrics FG by default in the tests. This is why I've included this.

Reply:
The test is marked as Serial already? I'd rather not enable something globally to avoid Serial just for a single test.

Reply:
This is the particular test I'm referring to: https://github.com/kubevirt/kubevirt/pull/12650/files#diff-5034f7c82b377dd518dd147554ad7df560b0cc667f1f0aa08e7a632d1436237aR471
It's just two tests, but if we disabled the feature globally, we need to make them Serial. I'm not against it, whatever we consider best.

Reply:
If the tests require it to be enabled but it is not by default, then they should be Serial.